Docker/qgis3.4.5/Dockerfile

```diff
@@ -74,16 +74,9 @@ RUN pip3 uninstall h5py -y
 ENV HDF5_DIR /usr/lib/x86_64-linux-gnu/hdf5/serial/
 RUN pip3 install --no-binary=h5py h5py
 
-RUN groupadd -g $GROUP_ID $GROUP_ID
-RUN useradd -m -l -d /home/$USERNAME --shell /bin/bash --uid $USER_ID --gid $GROUP_ID $USERNAME
-
 # Add the qgis-core plugins (like 'db_manager') to the PYTHONPATH
 ENV PYTHONPATH /usr/share/qgis/python/plugins
 
-COPY Docker/docker-entrypoint.sh /
-RUN chmod u+x /docker-entrypoint.sh
```

Comment:
The docker-entrypoint stuff has been replaced by the `USER nens` line at the bottom.